metadata/src/main/java/edu/unc/lib/dl/xml/FOXMLJDOMUtil.java

```diff
@@ -84,7 +85,7 @@ public String toString() {
     private static final String getDatastreamXpath = "/f:digitalObject/f:datastream[@ID='%DSID%']";
     private static final String labelXpath =
             "/f:digitalObject/f:objectProperties/f:property[@NAME='info:fedora/fedora-system:def/model#label']/@VALUE";
-    private static Logger log = Logger.getLogger(FOXMLJDOMUtil.class);
+    private static final Logger log = LoggerFactory.getLogger(FOXMLJDOMUtil.class);
```

Comment:
Does this still need to refer to FOXML?

Reply:
This class is going to be deleted, but I needed to clear out the log4j references in it in the meantime.